lang/py/src/avro/schema.py

```diff
@@ -815,21 +892,38 @@ def make_avsc_object(json_data, names=None):
     logical_type = None
     if 'logicalType' in json_data:
       logical_type = json_data.get('logicalType')
-      if logical_type != 'decimal':
+      if logical_type not in constants.SUPPORTED_LOGICAL_TYPE:
```

Comment:
Since it's checking if the logical type is in the supported logical type list, the `if 'logicalType' in json_data` in line #893 seems able to be skipped and just directly get the `logicalType` out of `json_data`. If it's not present in the `json_data`, it'd return `None`. Not a big deal though. :)